versions/3.1.1.md

```diff
@@ -1591,6 +1591,12 @@ content:
   application/octet-stream: {}
 ```
 
+```yaml
+# arbitrary JSON without constraints beyond being well-formed:
```

Comment:
I vaguely wonder if "being well-formed" will throw anyone.  Perhaps "being syntactically valid JSON"?  I am fine with it as it is, though.